recipe/conda_build_config.yaml

```diff
@@ -9,6 +9,8 @@ scalar:
 hdf5:
   - 1.10.6
   - 1.10.6=mpi*
+hdf5:
```

Comment:
Bad copy and paste, it should be `fftw`. Also, please add a blank line before.

Reply:
> Bad copy and paste, it should be `fftw`. Also, please add a blank line before.

thx.
Do I also need an equals sign (- = mpi*)? I'm just trying

Reply:
I think `fftw-mpi` is not correct, there is no such package in conda-forge. About the config spec, perhaps `*=mpi*` will do? Sorry, but I'm not really an expert, I'm here since the beginning, and rules have changed so much over time to make sense of things.